gensim/corpora/mmcorpus.py

```diff
@@ -19,21 +19,78 @@
 logger = logging.getLogger('gensim.corpora.mmcorpus')
 
 
-class MmCorpus(matutils.MmReader, IndexedCorpus):
+try:
+    # try to load fast, cythonized code if possible
+    from gensim.corpora._mmreader import MmReader
+    FAST_VERSION = 1
+    logger.info('Fast version of MmReader is being used')
```

Comment:
No need to log anything here (and after, because this will be shown for each import of gensim at all (because we import all stuff in `__init__.py`)

Reply:
ok, will remove